### PR TITLE
e2e: Skip Kommander cleanup at the end

### PR DESCRIPTION
**Pull request by dkoshkin**
Signed-off-by: Dimitri Koshkin <dimitri.koshkin@gmail.com>

**What type of PR is this?**

Chore

**What this PR does/ why we need it**:
After [recent kubeaddons changes](https://github.com/mesosphere/kubeaddons/pull/1062), the kubeaddons-controller will now fail on a delete error.
Currently, Kommander cannot be deleted, so removing it from the list of addons to cleanup.

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Checklist**

- [ ] The commit message explains the changes and why are needed.
- [ ] The code builds and passes lint/style checks locally.
- [ ] The relevant subset of integration tests pass locally.
- [ ] The core changes are covered by tests.
- [ ] The documentation is updated where needed.

### CODE REVIEW

**Review by samvantran — APPROVED** [review]

**Review by joejulian — APPROVED** [review]

**Comment by juliangieseke** [comment]
the linked JIRA was resolved, are we sure this isnt a regression?

**Review by gracedo — APPROVED** [review]

**Comment by gracedo** [comment]
We reused the JIRA but I believe the underlying issue is different. Perhaps we can create a new bug instead of reusing this one. Merging this to unblock PRs as tests won't pass until this is merged.

**Pull request merged**
